scripts/release/test/stage/setup/run.sh

```diff
@@ -1,26 +1,27 @@
 #!/usr/bin/env bash
 # shellcheck disable=2029
 
+set -ex
+
 # Path(s) are relative to the root of the Jenkins workspace.
 INSTANCE=$(cat scripts/release/common/ec2/tmp/instance)
-#BUCKET="$1"
 BRANCH="$1"
-BUILD_ENV=$(ssh -i ReleaseBuildInstanceKey.pem -o -A ubuntu@"$INSTANCE" cat build_env)
+
+aws s3 cp s3://algorand-devops-misc/tools/gnupg2.2.9_centos7_amd64.tar.bz2 .
+scp -i ReleaseBuildInstanceKey.pem -o StrictHostKeyChecking=no -r gnupg2.2.9_centos7_amd64.tar.bz2 ubuntu@"$INSTANCE":
+
+scp -i ReleaseBuildInstanceKey.pem -o StrictHostKeyChecking=no -r scripts/release/common/setup.sh ubuntu@"$INSTANCE":
+ssh -i ReleaseBuildInstanceKey.pem -A ubuntu@"$INSTANCE" bash setup.sh "$BRANCH"
+
+BUILD_ENV=$(ssh -i ReleaseBuildInstanceKey.pem -A ubuntu@"$INSTANCE" cat build_env)
 CHANNEL=$(sed -n 's/.*CHANNEL=\(.*\)/\1/p' <<< "$BUILD_ENV")
```

Comment:
The order of operations is essential here.  The `setup.sh` script *must* be copied to the instance and invoked before the remote call to `cat` the `build_env` file, which is created during `setup`.